grakn-test/src/test/java/ai/grakn/test/graql/reasoner/graphs/TestGraph.java

```diff
@@ -85,13 +84,15 @@ protected void loadGraqlFile(String fileName) {
         System.out.println("Loading " + fileName);
         if (fileName.isEmpty()) return;
 
-        QueryBuilder qb = graknGraph.graql();
+        GraknGraph graph = graph();
+        QueryBuilder qb = graph.graql();
         try {
             List<String> lines = Files.readAllLines(Paths.get(filePath + fileName), StandardCharsets.UTF_8);
             String query = lines.stream().reduce("", (s1, s2) -> s1 + "\n" + s2);
             qb.parse(query).execute();
+            graph.commit();
         }
-        catch (IOException e){
+        catch (IOException|GraknValidationException e){
```

Comment:
Kind of weird to have these exceptions being handled in the same manner. It's just a test though.